js/main.js

```diff
@@ -188,6 +189,7 @@ function wipe() {
   localStorage.setItem('entries', dataJSON);
 }
 
+$searchQuery.addEventListener('change', updateEntryView($searchQuery.value));
```

Comment:
What is this line doing? You are passing an argument to a function you are calling immediately (which incidentally that function definition has no named parameters or usage of this data that I can see), it looks like your callback would end up being `undefined` since `updateEntryView` doesn't return anything either